backend/internal/models/media.go

```diff
@@ -28,11 +28,13 @@ type Album struct {
 }
 
 type Track struct {
-	Media    MediaType
-	ID       int    `json:"id"`
-	AlbumID  int    `json:"album_id"`
-	Title    string `json:"title"`
-	Duration int    `json:"duration_seconds"`
+	Media       MediaType
+	ID          int       `json:"id"`
+	AlbumID     int       `json:"album_id"`
```

Comment:
lowkey can we add and populate AlbumName in here? I think we should keep the ID too so we can do more queries if needed but having the name for quick display would be nice

Reply:
I think we should also add artist names (and IDs) to both track and album, but we can save that for later since the multiple artist edge case makes the data kinda weird